pkg/glusterfs/controllerserver.go

```diff
@@ -183,56 +148,70 @@ func (cs *ControllerServer) validateCreateVolumeReq(req *csi.CreateVolumeRequest
 	return nil
 }
 
-func (cs *ControllerServer) checkExistingVolume(volumeName string, volSizeMB int) (string, []string, error) {
-	var (
-		tspServers  []string
-		mountServer string
-		err         error
-	)
+func (cs *ControllerServer) doVolumeCreate(volumeName string, volSizeMB int) error {
+	glog.V(4).Infof("Received request to create volume %s with size %d", volumeName, volSizeMB)
+	volMetaMap := make(map[string]string)
+	volMetaMap[volumeOwnerAnn] = glusterfsCSIDriverName
+	volumeReq := api.VolCreateReq{
+		Name:         volumeName,
+		Metadata:     volMetaMap,
+		ReplicaCount: defaultReplicaCount,
+		Size:         uint64(volSizeMB),
+	}
+
+	glog.V(2).Infof("volume create request: %+v", volumeReq)
+	volumeCreateResp, err := cs.client.VolumeCreate(volumeReq)
+	if err != nil {
+		glog.Errorf("failed to create volume: %v", err)
+		return status.Errorf(codes.Internal, "failed to create volume: %v", err)
+	}
 
+	glog.V(3).Infof("volume create response : %+v", volumeCreateResp)
```

Comment:
can this be a debug log?

Reply:
In what sense? Does being `V(3)` not make it a debug?

Reply:
am not sure about it :(

Reply:
You'd have to supply `-v3` to the command to get it to show up, so I say it counts.